homeworks/misha.kolosov_mwkolosov/homework_7/main.js

```diff
@@ -0,0 +1,17 @@
+const random = Math.floor(Math.random() * (100 - 50 + 1)) + 50;
+
+const fib = (maxNum) => {
+    const arr = [0, 1];
+    for (let i = 0; i < 20; i++) {
```

Comment:
why limiting your loop with only 20 iterations? how do you know the exact amount of iterations needed?

I would better wright a `while` loop with a `break` condition, it would be more logical to push to `arr` while fibo number is less than `maxNum`